### ISSUE TITLE
Spectral Deconvolution error, "flist" is null even with verified feature list

### ISSUE THREAD

**Issue by LoopyFig** [mzmine/mzmine3#728]
### Basic information

* My operating system: Windows 10
* My MZmine version: MZmine 3.0.21-beta

### What happened

* Both modules of Spectral Deconvolution are non-functional for me
* when I attempt to start the program, after running for a bit it produces the following error:
* 
![image](https://user-images.githubusercontent.com/64052124/173117291-cd0dbd27-602d-4483-8d1b-058c70acc347.png)

* additionally the preview function doesn't actually display anything:
* 
![image](https://user-images.githubusercontent.com/64052124/173117126-1c056d19-e163-48e5-8ede-6796bdd4152b.png)

* as additional information, I am following the suggested GCMS workflow. the ADAP feature resolver seemed to run without issue, and when I display the feature list it looks like this:
* ![image](https://user-images.githubusercontent.com/64052124/173118553-71cc6bbc-634c-4b35-ac6f-52732e528980.png)

* the summary of the process looks like this, I'm noticing it looks like there's no listed feature list as an input, so is this potentially the issue?
* 
![image](https://user-images.githubusercontent.com/64052124/173118717-ec359cc9-8dde-4724-a7d9-9907b2058572.png)

**Comment by LoopyFig** [comment]
if i upload an mzxml file and a batch file, would that help in resolving this issue?

**Comment by LoopyFig** [comment]
as an update, the spectral deconvolution error occurs regardless of whether the input comes from the ADAP peak detection or local minimum peak detection

**Comment by SteffenHeu** [comment]
Hi LoopyFig,

thanks for reporting. 
As far as i know, @asmirn1 is already working on a fix.

**Comment by asmirn1** [comment]
@robinschmid @SteffenHeu I've made some changes to my fork of MZmine 3 to make the spectral deconvolution work. I'll check it and make a pull request.

**Comment by nic-Oban** [comment]
Hi @asmirn1,
I'm currently testing MZmine3, trying to switch from MZmine2, and I face the same issue that @LoopyFig.
I'm running MZmine 3.1.0 portable (even tried pre-Release 3.1.1) with Win10.
Let me know if I can help you in debugging this problem and thanks a lot for your beautiful work to all of the contributors!!!
Have a good day

**Comment by robinschmid** [comment]
@LoopyFig @nic-Oban can you test the latest development build. We just merged a fix into version 3.1.3
https://github.com/mzmine/mzmine3/releases

Please report if this fixes both methods for your data.

**Comment by nic-Oban** [comment]
Hi @robinschmid ,
thanks a lot for this release!
I just tested it on a dataset I know well and Multivariate curve resolution module is now working as expected I think. However, the preview is not available which make it still unusable on unknown chromatograms.
For Hierarchical clustering, I didn't used it much but it still throwing an error ('Unknown error' during peak decomposition. Cannot invoke "io.github.mzmine.datamodel.FeatureInformation.getAllPropoerties()" because the return value of "io.github.mzmine.datamodel.feature.FeatureListRow.getFeatureInformation()" is null) and no preview too.
Yesterday, I tested 3.1.1 release on an Ubuntu OS and I faced the same errors.
Thanks again for this fix and let me know if I can do anything more to help you.
Have a good day.
